teku/src/main/java/tech/pegasys/teku/cli/options/NetworkOptions.java

```diff
@@ -25,11 +25,11 @@
   private String network = "medalla";
 
   @Option(
-      names = {"--initial-state"},
+      names = {"--genesis-state", "--initial-state"},
```

Comment:
I wonder if we should leave this as `--initial-state` because ultimately we want to be able to sync just from a state and so would wind up being able to just specify that state here.

Even now if the slot of the initial state was 0 we can generate the block and if it's not, we require the block to be passed in via the extra arg.

However it winds up, I think I'd avoid making changes to the stable CLI options before we know exactly where it's going to land.

Reply:
Yeah, I was initially thinking they would merge, but then it just seemed simpler to keep them separate.  But it probably is premature to rename right now.  Reverted this.